dp_branding/index.php

```diff
@@ -0,0 +1,158 @@
+<?php
+$relPath = '../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'theme.inc');
+
+/*
+Author: cpeel
+Created: 2016-05-03
+*/
+
+require_login();
```

Comment:
This page needs to remain accessible to non-logged-in-users who want to use branding assets to link to our site.

Reply:
Fixed.